src/network/wait-online/manager.c

```diff
@@ -37,6 +37,13 @@ static int manager_link_is_online(Manager *m, Link *l, LinkOperationalStateRange
          *       0: operstate is not enough
          *       1: online */
 
+        assert(m);
+        assert(l);
+
+        AddressFamily required_family;
+        bool needs_ipv4;
+        bool needs_ipv6;
+
```

Comment:
We usually define functions in the following order,
1. variable declarations,
2. assertions,
3. comments.

So, please move this above the comments, and first declare the variables.